src/main/webapp/stylesheets/teammates.css

```diff
@@ -215,11 +215,35 @@ h2.subcaption {
     display: block;
 }
 
+h2.subcaptionTypeTwo {
+    color: #4379D1;
+    font-size: 15px;
+    font-family: Arial, Helvetica, sans-serif;
+    font-weight: bold;
+    margin: 10px auto;
+    display: block;
+    text-align:center;
+}
```

Comment:
This is almost identical to `h2.subcaption` above. Just use two classes `subcaption align-center` to achieve the same effect.